weekly/timetozingg.txt

```diff
@@ -0,0 +1,13 @@
+article:
+  author:
+    linkedin: "linkedin.com/in/sonalgoyal/"
+    name: "Zingg.ai"
+    twitter: "@sonalgoyal"
```

Comment:
Hi Sonal, Thanks for the contribution. I've updated the guidelines for the contribution here.


```
The founder's story will be published every Wednesday as a separate edition.
Open a pull request with a simple .md file with a title and a one-pager about what you're working.
You could also point to a blog post you already published, given that the data engineering weekly can republish the content.
```

How about Data Engineering Weekly republish the blog post that was shared instead of a link?

Reply:
Yes I am game for republish. Do you need me to change the pull request to an md file instead of txt ?

Reply:
Thank you, yeh text file or as a markdown file so that I can copy the same header formatting into the newsletter

Reply:
Sure. timetozingg.txt is already txt. do you need any changes to the pr?

Reply:
timetozingg.txt  works. since the blog already contains the formatting I can follow the same. Let me publish this tomorrow. 
Thank you for adding your story.